x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -498,81 +592,83 @@ export const UserProfile: FunctionComponent<UserProfileProps> = ({ user, data })
-              <UserPasswordEditor
-                user={user}
-                onShowPasswordForm={() => setShowChangePasswordForm(true)}
+            {showChangePasswordForm ? (
+              <ChangePasswordModal
+                username={user.username}
+                onCancel={() => setShowChangePasswordForm(false)}
+                onSuccess={() => setShowChangePasswordForm(false)}
               />
-            </Form>
-            <EuiSpacer />
-          </EuiPageTemplate>
-        </Breadcrumb>
-      </FormChangesProvider>
-    </FormikProvider>
+            ) : null}
+
+            <EuiPageTemplate
+              className="eui-fullHeight"
+              pageHeader={{
+                pageTitle: (
+                  <FormattedMessage
+                    id="xpack.security.accountManagement.userProfile.title"
+                    defaultMessage="Profile"
+                  />
+                ),
+                pageTitleProps: { id: titleId },
+                rightSideItems: rightSideItems.reverse().map((item) => (
+                  <EuiDescriptionList
+                    textStyle="reverse"
+                    listItems={[
+                      {
+                        title: (
+                          <EuiText color={euiTheme.colors.darkestShade} size="s">
+                            <EuiFlexGroup responsive={false} alignItems="center" gutterSize="none">
+                              <EuiFlexItem grow={false}>{item.title}</EuiFlexItem>
+                              <EuiFlexItem grow={false} style={{ marginLeft: '0.33em' }}>
+                                <EuiIconTip type="questionInCircle" content={item.helpText} />
+                              </EuiFlexItem>
+                            </EuiFlexGroup>
+                          </EuiText>
+                        ),
+                        description: (
+                          <span data-test-subj={item.testSubj}>
+                            {item.description || (
+                              <EuiText color={euiTheme.colors.disabledText} size="s">
+                                <FormattedMessage
+                                  id="xpack.security.accountManagement.userProfile.noneProvided"
+                                  defaultMessage="None provided"
+                                />
+                              </EuiText>
+                            )}
+                          </span>
+                        ),
+                      },
+                    ]}
+                    compressed
+                  />
+                )),
+              }}
+              bottomBar={formChanges.count > 0 ? <SaveChangesBottomBar /> : null}
+              bottomBarProps={{ paddingSize: 'm', position: 'fixed' }}
+              restrictWidth={1000}
+            >
+              <Form aria-labelledby={titleId}>
+                <UserDetailsEditor user={user} />
+                {isCloudUser ? null : <UserSettingsEditor formik={formik} />}
```

Comment:
The placement of theme selection above all other current user profile form elements feels odd to me. We originally had it in mind for the theme settings (and other groups of user settings) to be on separate settings pages. But that probably wouldn't make sense to do so until we have more settings added/transitioned. Perhaps for now we could simply move this to the bottom, below the change password section. Thoughts?

Reply:
Right since it was only one setting, it felt a little out of place/too much overhead to have its own tab on UserProfiles.

I can certainly move it to the bottom of the form though